src/components/LHNOptionsList/OptionRowLHN.js

```diff
@@ -52,14 +60,24 @@ const defaultProps = {
     onSelectRow: () => {},
     isFocused: false,
     style: null,
+    comment: '',
 };
 
 function OptionRowLHN(props) {
     const optionItem = SidebarUtils.getOptionData(props.reportID);
 
-    if (!optionItem) {
-        return null;
-    }
+    const runUseEffectOnlyOnce = useCallback(() => {
+        if (!optionItem) {
+            return null;
+        }
```

Comment:
I'm afraid that's not right. We will crash the app if `optionItem` is `null`—the code before these changes was meant to return `null` early for the component.

Reply:
resolved here https://github.com/Expensify/App/pull/20730/commits/68167d9599303b8775861101eb726d656142ffc8